src/ViewController.js

```diff
@@ -233,26 +245,24 @@ class ViewController extends HasModels {
       return ret
     })    
     .filter((k) => {
-      if(display.length > 0)
-        return display.includes(k.name)
-      else
-        return true
-    })
-    .filter((k) => {
       let ret = ignore.includes(k.name) 
       if(!ret) ret = ignoreType.includes(k.type)
       return !ret
     })
+    if (display.length > 0) {
```

Comment:
Is the idea that this lets `displayFields` set the display order? If so, might make sure that behavior is documented.